pytroll_collectors/__init__.py

```diff
@@ -21,5 +21,3 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 """The place where everything starts :)."""
-from . import _version
```

Comment:
This means there is no `pytroll_collectors.__version__` though right? I don't use this package but for consistency I'd like this re-added.